puppetboard/templates/layout.html

```diff
@@ -56,8 +60,68 @@
           $.getScript('{{url_for('static', filename='js/tables.js')}}')
           {% block onload_script %} {% endblock onload_script %}
         })
+
+    
+      $('#export').click(function() {
+      var titles = [];
+      var data = [];
+
+  $('.dataTable th').each(function() {
```

Comment:
Moving my comment about indentation, as this where you stopped indenting.

If line 67 is indented and no block has ended this should also be indented. 

Also another small thing lines 66 and 67 should have a 2 space indent as children of the function starting on 65.